bc/core/utils/tests/base.py

```diff
@@ -0,0 +1,3 @@
+from faker import Faker
+
+faker = Faker()
```

Comment:
Why do this? Why not just do `faker = Faker()` in your tests?

Reply:
I found the recommendation in this [article](https://www.hacksoft.io/blog/improve-your-tests-django-fakes-and-factories) from the [django-guidestyle](https://github.com/HackSoftware/Django-Styleguide#how-to-ask-a-question-or-propose-something). it says: 

> We highly recommend "proxying" the Faker instance and using it in your app that way.
> 
> You'd most likely want to have the same configuration when you use fakes around your app. Same goes if you want to customize the providers and use them in different places.

I also checked the [Django-Styleguide-Example](https://github.com/HackSoftware/Django-Styleguide-Example) repo and They're still using this approach to create a [Faker](https://faker.readthedocs.io/en/master/index.html) instance.